pressbooks/pressbooks
# Upgrade Docraptor / PrinceXML

Issue #2228 

This PR upgrades [docraptor/docraptor](https://github.com/DocRaptor/docraptor-php) to v2 and changes the pipeline to use the latest version of PrinceXML.

For now, [guzzlehttp/guzzle](https://github.com/guzzle/guzzle) needs to be locked on version ^6.2 because docraptor has no support for ^7.0. See https://github.com/DocRaptor/docraptor-php/issues/23 and https://github.com/DocRaptor/docraptor-php/pull/24

#### How to test

1. Generate a book export and a book cover export using the dev branch.
1. Switch to this branch and `composer install` to update dependencies.
1. Generate new book and cover exports.
1. Both export versions should be equal.
